website/source/guides/security/vault-pki-integration.html.md

```diff
@@ -0,0 +1,550 @@
+---
+layout: "guides"
+page_title: "Vault PKI Secrets Engine Integration"
+sidebar_current: "guides-security-vault-pki"
+description: |-
+  Securing Nomad's cluster communication with TLS is important for both
+  security and easing operations. Nomad can use mutual TLS (mTLS) for
+  authenticating for all HTTP and RPC communication. This guide will leverage
+  Vault's PKI secrets engine to accomplish this task.
+---
+
+# Vault PKI Secrets Engine Integration
+
+You can use [Consul Template][consul-template] in your Nomad cluster to
+integrate with Vault's [PKI Secrets Engine][pki-engine] to generate and renew
+dynamic X.509 certificates. By using this method, you enable each node to have a
+unique certificate, eliminating sharing and the accompanying pain of revocation
+and rollover. You can also keep certificate TTLs relatively short which makes
```

Comment:
We don't even support revocation lists: #4900 

Short TTLs are the only way we support removing a node from a cluster via TLS.